core/src/test/java/com/google/errorprone/refaster/testdata/template/TypeArgumentsMethodInvocationTemplate.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 The Error Prone Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.refaster.testdata.template;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+
+/** Sample template for introducing type arguments on a method invocation in an @AfterTemplate. */
+public class TypeArgumentsMethodInvocationTemplate {
+  @BeforeTemplate
+  boolean before(String string) {
+    return string.isEmpty();
+  }
+
+  @AfterTemplate
+  boolean after(String string) {
+    return string.<String>isEmpty();
```

Comment:
Discussed offline: surprisingly this is valid code (TIL!), but let's use a realistic example, one with a method that actually accepts a generic type parameter. (For example, `ExecutorService#submit`.)

Reply:
Yes, just pushed the update, now it uses a realistic example.